llvm/test/CodeGen/AVR/issue-132203.ll

```diff
@@ -0,0 +1,42 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple=avr -mattr=+movw | FileCheck %s --check-prefix=MOVW
+; RUN: llc < %s -mtriple=avr -mattr=-movw | FileCheck %s --check-prefix=NOMOVW
+
+define i16 @food(ptr %this) {
```

Comment:
I think it'd be nice to have both this original snippet and the minimized one (the latter is easier to debug and play around with, while the former allows us to make sure the underlying issue was actually solved).

Reply:
Your simplified test has been added. Thanks!